johnsonandjohnson/Bodiless-JS
# Add product listing template

### Changes
- Add product listing page template
- Implement flexbox with just one product variation to select
- Add product organism with BvReview field
- Change placeholder image for header
- Fix several flexbox issues

### Dependencies
- It is expected that github.com/johnsonandjohnson/Bodiless-JS/pull/9 is merged

### How to test
- open /products page
- upload header image
- click at the bottom area of the page - the plus (+) icon should appear at the left menu
- add some products
- try to replace, edit title, image, body, and links
